Formula/pocsuite3.rb

```diff
@@ -0,0 +1,138 @@
+class Pocsuite3 < Formula
+  include Language::Python::Virtualenv
+
+  desc "Open-sourced remote vulnerability testing framework"
+  homepage "https://pocsuite.org/"
+  url "https://files.pythonhosted.org/packages/5d/b6/3084cf84a7b012a629c3098cea5c887c0d3976057e31c2e92287024d8811/pocsuite3-1.8.7.tar.gz"
+  sha256 "fb43b4d0684390bdffe38c8d56714a5635e405ce7946c1b186e78cc7d224287c"
+  license "GPL-2.0-only"
+  head "https://github.com/knownsec/pocsuite3.git", branch: "master"
+
+  depends_on "rust" => :build # for cryptography
+  depends_on "openssl@1.1"
+  depends_on "python@3.9"
```

Comment:
```suggestion
  depends_on "python@3.10"
```
Why not Python 3.10?

Reply:
Python 3.10 was released on October 4, 2021. It is too new and may have some compatibility issues.
eg, https://github.com/knownsec/pocsuite3/pull/230